### DISCUSSION TITLE
Porque os elementos desse Array não estão sendo separados por vírgula?

### DISCUSSION THREAD

**Discussion by leomalm** [frontendbr/forum#2396] [Perguntas e Respostas] [answered]
Pessoal, estou com um problema com este código:
```js
  function listaNum (numero) {
    var num = [];
    var i = 0;
    while (i <= numero) {
      num += i;
      i++
    }
    console.log(num.length);
    return num;
  }
```
Sempre que a lista de números naturais menores ou iguais ao número inserido pelo usuário na função é gerada , o programa retorna a lista de elementos sem separa-los por vírgula. Inclusive, quando peço para o programa imprimir num.length, ele informa um total considerando cada algarismo como um índice, p. ex., ao imprimir num.length para a lista '1234567891011', o programa informa 13 em vez de 11.

Podem me ajudar?

**Comment by felquis** [comment, marked as answer]
Em vez de `num += i`
O que vc quer é `num.push(i)`

**Reply by leomalm** [reply]
Perfeito, felquis. Muito obrigado!
